NuGet.config

```diff
@@ -23,11 +25,21 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-extensions -->
+    <add key="darc-int-dotnet-extensions-c7c07d5" value="true" />
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
     <add key="darc-int-dotnet-extensions-4210e7d" value="true" />
     <add key="darc-int-dotnet-extensions-4210e7d" value="true" />
     <!--  End: Package sources from dotnet-extensions -->
     <!--  Begin: Package sources from dotnet-core-setup -->
+    <add key="darc-int-dotnet-core-setup-df8abc0" value="true" />
+    <add key="darc-int-dotnet-core-setup-ff13fbc" value="true" />
+    <add key="darc-int-dotnet-core-setup-e365b58" value="true" />
+    <add key="darc-int-dotnet-core-setup-7b9e303" value="true" />
+    <add key="darc-int-dotnet-core-setup-a80e00a" value="true" />
+    <add key="darc-int-dotnet-core-setup-abd1c7b" value="true" />
+    <add key="darc-int-dotnet-core-setup-d05fcb2" value="true" />
+    <add key="darc-int-dotnet-core-setup-d05fcb2" value="true" />
+    <add key="darc-int-dotnet-core-setup-d05fcb2" value="true" />
```

Comment:
@mmitche what's with the duplicates here❔ I have see comments about the `-1`, `-2` etc. oddities but this seems even weirder. (Bit more further down.)